# Copyright Materialize, Inc. and contributors. All rights reserved.
#
# Use of this software is governed by the Business Source License
# included in the LICENSE file at the root of this repository.
#
# As of the Change Date specified in that file, in accordance with
# the Business Source License, use of this software will be governed
# by the Apache License, Version 2.0.

diff --git a/internal/disk/fdatasync_linux.go b/internal/disk/fdatasync_linux.go
index 4d3986773..b3b6b1505 100644
--- a/internal/disk/fdatasync_linux.go
+++ b/internal/disk/fdatasync_linux.go
@@ -22,7 +22,6 @@ package disk
 
 import (
 	"os"
-	"syscall"

 	"golang.org/x/sys/unix"
 )
@@ -38,7 +37,7 @@ import (
 // The aim of fdatasync() is to reduce disk activity for applications that
 // do not require all metadata to be synchronized with the disk.
 func Fdatasync(f *os.File) error {
-	return syscall.Fdatasync(int(f.Fd()))
+	return nil
 }
 
 // FadviseDontNeed invalidates page-cache
